pkgs/development/tools/misc/binutils/default.nix

```diff
@@ -53,6 +53,14 @@ stdenv.mkDerivation rec {
     # elf32-littlearm-vxworks in favor of the first.
     # https://github.com/NixOS/nixpkgs/pull/30484#issuecomment-345472766
     ./disambiguate-arm-targets.patch
+
```

Comment:
I or @bgamari should email upstream cause the default behavior is insane.

Reply:
I have reported this at https://sourceware.org/bugzilla/show_bug.cgi?id=22458, with no response so far. (Said so in https://github.com/NixOS/nixpkgs/pull/30484#issuecomment-345491222.)

Reply:
Ah, you meant `always-search-rpath.patch` below.